README.md

```diff
@@ -111,6 +112,45 @@ Eligible students can apply for Heroku platform credits through [Heroku for GitH
 
 You may want to read about [configuring optional features](#configure-optional-features).
 
+## Deploy to own servers with Kamal
+
+[Kamal](https://kamal-deploy.org/) offers zero-downtime deploys, rolling restarts, asset bridging, remote builds, accessory service management, and everything else you need to deploy and manage your web app in production with Docker. Originally built for Rails apps, Kamal will work with any type of web app that can be containerized.
+
+First, create your production credentials file.
+
+```plain
+bin/rails credentials:edit --environment production
+```
+
+Next, uncomment the `database:` section
+
+```yaml
+database:
+  password: some-long-string
+```
+
+Second, create a Docker Hub access token and store it as local env var `KAMAL_REGISTRY_PASSWORD`.
```

Comment:
I know github lets you host containers. I was using that for a different github project. If I were to enable github container registery for HostedGPT would that save people the trouble of needing to create a Docker Hub account? It must be that Kamal needs to pull from a docker container registery? (I need to watch the Kamal tutorial video)

Reply:
I think Kamal likes to build a new image and deploy it; whereas docker compose supports running pre-built containers or re-building them.

Specifically:

* the `servers` processes are going to run based on the `image` that is built from the local src project
* the `accessories` processes are run based on existing images.

So each person who wants to use Kamal will be building their own image to their own docker registry (docker hub, or wherever).